alts/shared/models.py

```diff
@@ -205,6 +205,8 @@ def __init__(self, **data):
 
     # Whether to setup Celery SSL
     use_ssl: bool = False
+    # SSL configuration section
+    ssl_config: typing.Optional[SslConfig]
```

Comment:
Please add `= None`
In a new version of pydentic `Optional` field should have a default value https://github.com/pydantic/pydantic/discussions/2353

Reply:
Fixed